lib/components/home/artwork_rail.js

```diff
@@ -96,9 +101,12 @@ class ArtworkRail extends React.Component {
 
 const styles = StyleSheet.create({
   title: {
-    margin: 10,
-    fontSize: 20,
+    margin: isPad ? 40 : 20,
```

Comment:
An option, could be to create our own subclass of StyleSheet ( or our own function, whatever)

```
const styles = StyleSheet.create({
  title: {
    margin: 20,
    alignSelf: 'center',
    textAlign: 'center',
  },
  container: {
    backgroundColor: 'white',
  }
}, largeDevice: {
  title: {
    margin: 40
  } 
})
```
 
which then overwrites the default with the large one on bigger devices